src/librustc_yk_sections/emit_tir.rs

```diff
@@ -264,6 +267,8 @@ impl<'tcx> ToPack<ykpack::Terminator> for (&ConvCx<'_, 'tcx, '_>, &Terminator<'t
                     unwind_bb: unwind_bb.map(|bb| u32::from(bb)),
                 },
             TerminatorKind::Call{ref func, cleanup: cleanup_bb, ref destination, .. } => {
+                // In MIR, a call instruction accepts an arbitrary operand, but in TIR we special
```

Comment:
Not sure I understand this (I'm not saying it's badly phrased... I just don't understand it!).

Reply:
In mir they have `Call(Operand)` where `Operand` is really general. It can express things which I don't think you can even call.

Reply:
Ah, I see.